src/components/ResponsiveAppBar.jsx

```diff
@@ -126,10 +127,19 @@ const ResponsiveAppBar = () => {
               display: { xs: "none", md: "flex" },
               flexDirection: "row",
               alignItems: "center",
+              width: '100%',
+              justifyContent: 'space-between'
```

Comment:
A hack to add some space is to set an empty `div` with flexGrowth = 1
```jsx
<Box sx={{flexGrowth: 1}}></Box>
```

This div will extend itself to fill all the remaining space